exercises/Functions-6-R.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: exercise
+title: Functions 6
+language: R
+---
+
+This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
+
+A colleague has produced a file with one DNA sequence on each line. Download
+[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
+`read.csv()`. The file has no header and is separated by white space (`’’`).
```

Comment:
>> `read.csv("dna_sequences_1.txt", header=FALSE)` seems to work

For me too. I added the sep = '' as a fix somewhere, but it's clearly not
needed here.
Obviously, `sep = ','` is used a ton.

On Mon, May 4, 2015 at 5:11 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Functions-6-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/36#discussion_r29629090>
> :
>
> > @@ -0,0 +1,22 @@
> > +---
> > +layout: exercise
> > +title: Functions 6
> > +language: R
> > +---
> > +
> > +This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
> > +
> > +A colleague has produced a file with one DNA sequence on each line. Download
> > +[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
> > +`read.csv()`. The file has no header and is separated by white space (`’’`).
>
> Why specify a separator at all? read.csv("dna_sequences_1.txt",
> header=FALSE) seems to work fine for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/36/files#r29629090>
> .
>